components/Column.vue

```diff
@@ -0,0 +1,45 @@
+<script setup lang="ts">
+import { VueDraggable } from 'vue-draggable-plus'
+import type { Issue, IssueStatus } from '../constants/issues'
+
+const props = defineProps<{ title: IssueStatus; issues: Issue[] }>()
+const issuesModel = ref(structuredClone(toRaw(props)).issues) // create new ref with the same contents
+
+const STATUS_TO_ICON_MAP = {
```

Comment:
FYI Radicle already has a term called IssueStatus which can be `"open"` or `"closed"`. So we should (in this PR or eventually in another one) move away from overloading the term Status with our "Todo|Doing|Done|etc" which are supposed to be inferred after scanning the *labels* instead.

Reply:
Good point, let's sync and I'll address it in an upcoming PR.